website/src/css/_admonitions.scss

```diff
@@ -1,5 +1,5 @@
 .alert {
-  @include detox-colors(--detox-color-a0);
+  @include detox-colors-light(--detox-color-a0);
```

Comment:
I don't see a reason to include colors as a mixin here.

Reply:
Thanks. Removed and modified the admonitions based on the changes in the _colors.scss file map() function.